### ISSUE TITLE
Weights of trained net

### ISSUE THREAD

**Issue by alevangel** [DeepMicroscopy/MIDOGpp#6]
Where I can donwload the weights of the pretrained nets?

**Comment by alevangel** [comment]
@maubreville is not possibile to upload it on an external website?

**Comment by FraukeWilm** [comment]
Hi, we have decided to upload the weights for the best-performing model trained on all domains [here](http://deepmicroscopy.org/files/midogpp.pth). Please be aware that the results reported in the manuscript are the average results of a five-fold cross-validation so the performance on the test set might slightly differ from the reported values.

**Comment by alevangel** [comment]
Thank you very much. I will take it into consideration.

**Comment by alevangel** [comment]
@FraukeWilm linked to the weights, can you also provide the x-validation configuration you used to perform the evaluation?

```
x-validation:
  train: None
  valid: None
```

**Comment by FraukeWilm** [comment]
train: '["001.tiff", "002.tiff", "003.tiff", "004.tiff", "005.tiff", "011.tiff",
      "012.tiff", "013.tiff", "016.tiff", "017.tiff", "019.tiff", "020.tiff", "021.tiff",
      "022.tiff", "023.tiff", "024.tiff", "025.tiff", "030.tiff", "034.tiff", "036.tiff",
      "038.tiff", "039.tiff", "041.tiff", "042.tiff", "043.tiff", "045.tiff", "047.tiff",
      "048.tiff", "049.tiff", "051.tiff", "053.tiff", "056.tiff", "060.tiff", "061.tiff",
      "062.tiff", "063.tiff", "065.tiff", "067.tiff", "068.tiff", "069.tiff", "070.tiff",
      "071.tiff", "073.tiff", "075.tiff", "077.tiff", "078.tiff", "080.tiff", "081.tiff",
      "082.tiff", "083.tiff", "085.tiff", "086.tiff", "087.tiff", "089.tiff", "090.tiff",
      "091.tiff", "092.tiff", "094.tiff", "096.tiff", "098.tiff", "099.tiff", "101.tiff",
      "103.tiff", "105.tiff", "106.tiff", "108.tiff", "110.tiff", "117.tiff", "119.tiff",
      "121.tiff", "122.tiff", "125.tiff", "127.tiff", "129.tiff", "130.tiff", "132.tiff",
      "133.tiff", "134.tiff", "135.tiff", "136.tiff", "138.tiff", "140.tiff", "141.tiff",
      "143.tiff", "144.tiff", "145.tiff", "146.tiff", "149.tiff", "201.tiff", "204.tiff",
      "205.tiff", "206.tiff", "207.tiff", "208.tiff", "209.tiff", "210.tiff", "211.tiff",
      "213.tiff", "214.tiff", "215.tiff", "216.tiff", "217.tiff", "221.tiff", "222.tiff",
      "223.tiff", "224.tiff", "226.tiff", "227.tiff", "228.tiff", "229.tiff", "231.tiff",
      "233.tiff", "235.tiff", "237.tiff", "240.tiff", "242.tiff", "244.tiff", "245.tiff",
      "246.tiff", "248.tiff", "250.tiff", "255.tiff", "258.tiff", "259.tiff", "260.tiff",
      "261.tiff", "262.tiff", "263.tiff", "264.tiff", "265.tiff", "266.tiff", "271.tiff",
      "272.tiff", "273.tiff", "274.tiff", "277.tiff", "280.tiff", "281.tiff", "282.tiff",
      "284.tiff", "285.tiff", "287.tiff", "288.tiff", "289.tiff", "290.tiff", "293.tiff",
      "294.tiff", "296.tiff", "297.tiff", "298.tiff", "299.tiff", "300.tiff", "301.tiff",
      "302.tiff", "303.tiff", "306.tiff", "307.tiff", "308.tiff", "309.tiff", "312.tiff",
      "314.tiff", "316.tiff", "318.tiff", "319.tiff", "321.tiff", "323.tiff", "324.tiff",
      "325.tiff", "326.tiff", "327.tiff", "328.tiff", "330.tiff", "332.tiff", "333.tiff",
      "334.tiff", "335.tiff", "337.tiff", "340.tiff", "341.tiff", "342.tiff", "347.tiff",
      "349.tiff", "350.tiff", "351.tiff", "352.tiff", "356.tiff", "357.tiff", "358.tiff",
      "359.tiff", "360.tiff", "361.tiff", "362.tiff", "363.tiff", "364.tiff", "365.tiff",
      "367.tiff", "369.tiff", "372.tiff", "374.tiff", "376.tiff", "377.tiff", "378.tiff",
      "379.tiff", "380.tiff", "381.tiff", "382.tiff", "384.tiff", "387.tiff", "391.tiff",
      "392.tiff", "393.tiff", "394.tiff", "395.tiff", "396.tiff", "397.tiff", "399.tiff",
      "402.tiff", "403.tiff", "404.tiff", "405.tiff", "407.tiff", "408.tiff", "409.tiff",
      "412.tiff", "415.tiff", "416.tiff", "418.tiff", "420.tiff", "423.tiff", "424.tiff",
      "428.tiff", "429.tiff", "431.tiff", "432.tiff", "433.tiff", "434.tiff", "435.tiff",
      "436.tiff", "437.tiff", "440.tiff", "441.tiff", "443.tiff", "444.tiff", "445.tiff",
      "446.tiff", "447.tiff", "448.tiff", "450.tiff", "451.tiff", "455.tiff", "459.tiff",
      "460.tiff", "461.tiff", "462.tiff", "465.tiff", "466.tiff", "467.tiff", "468.tiff",
      "469.tiff", "470.tiff", "471.tiff", "473.tiff", "475.tiff", "478.tiff", "480.tiff",
      "482.tiff", "483.tiff", "486.tiff", "488.tiff", "489.tiff", "492.tiff", "493.tiff",
      "494.tiff", "495.tiff", "496.tiff", "499.tiff", "500.tiff", "501.tiff", "502.tiff",
      "503.tiff", "505.tiff", "506.tiff", "507.tiff", "508.tiff", "509.tiff", "511.tiff",
      "513.tiff", "514.tiff", "515.tiff", "516.tiff", "517.tiff", "518.tiff", "519.tiff",
      "520.tiff", "521.tiff", "522.tiff", "524.tiff", "527.tiff", "529.tiff", "530.tiff",
      "532.tiff", "536.tiff", "542.tiff", "543.tiff", "544.tiff", "545.tiff", "546.tiff",
      "547.tiff", "548.tiff", "549.tiff", "550.tiff", "551.tiff", "553.tiff"]'
    valid: '["008.tiff", "009.tiff", "010.tiff", "027.tiff", "028.tiff", "033.tiff",
      "037.tiff", "040.tiff", "044.tiff", "046.tiff", "054.tiff", "055.tiff", "072.tiff",
      "076.tiff", "079.tiff", "084.tiff", "097.tiff", "111.tiff", "112.tiff", "113.tiff",
      "114.tiff", "118.tiff", "120.tiff", "126.tiff", "131.tiff", "137.tiff", "139.tiff",
      "142.tiff", "148.tiff", "202.tiff", "219.tiff", "225.tiff", "238.tiff", "243.tiff",
      "247.tiff", "254.tiff", "267.tiff", "270.tiff", "275.tiff", "278.tiff", "279.tiff",
      "283.tiff", "286.tiff", "304.tiff", "313.tiff", "317.tiff", "320.tiff", "322.tiff",
      "331.tiff", "343.tiff", "348.tiff", "355.tiff", "371.tiff", "385.tiff", "388.tiff",
      "400.tiff", "401.tiff", "406.tiff", "410.tiff", "411.tiff", "417.tiff", "421.tiff",
      "430.tiff", "438.tiff", "449.tiff", "452.tiff", "453.tiff", "454.tiff", "457.tiff",
      "464.tiff", "474.tiff", "479.tiff", "490.tiff", "498.tiff", "523.tiff", "525.tiff",
      "526.tiff", "535.tiff", "540.tiff"]'

**Comment by maubreville** [comment]
I just checked and the link should still be valid. Could you tell me what error message you get?

**Comment by salbahra** [comment]
Ah, thank you for confirming. It turns out my browser (Chrome) was auto rejecting it however opening the downloads tab showed a `Keep` button. Thanks again!

**Comment by maubreville** [comment]
ok, perfect. Then I'm closing the issue :-)

**Comment by shikha-gist** [comment]
Hi , the list you provided here for "x-validation:
  train:  (the list you have given above)
  valid:   (the list you have given above)
" is fixed? I thought you have used the 5 fold validation method in the code, could you please clarify this list? would we update this in the "all.yaml" file given in the "config" folder?

**Comment by fwilm-mira** [comment]
Hi, thank you for your question. The list above is the train and validation split that was used for training the model of which we made the weights publicly available. If you train a model yourself, the split will depend on the fold that you are using.

**Comment by FraukeWilm** [comment]
Hi, thank you for your question. The list above is the train and validation split that was used for training the model of which we made the weights publicly available. If you train a model yourself, the split will depend on the fold that you are using.
